src/biokbase/narrative/jobs/jobcomm.py

```diff
@@ -282,32 +293,42 @@ def _cancel_job(self, req: JobRequest) -> None:
             raise
         self._lookup_job_state(req)
 
-    def _retry_job(self, req: JobRequest) -> None:
+    def _retry_jobs(self, req: JobRequest) -> None:
         """
-        In this case, retry_results is formatted as:
-        {
-            "job_id": original_job_id,
-            "retry_id": new_job_id
-        }
+        retry_results is formatted as:
+        [
+            {
+                "job_id": original_job_id,
+                "retry_id": new_job_id
+            },
+            ...
+        ]
         """
         self._verify_job_id(req)
         try:
-            retry_results = self._jm.retry_job(req.job_id)
-            self.send_comm_message("job_retried", retry_results)
-            self.send_comm_message("new_job", {"job_id": retry_results["retry_id"]})
-        except ValueError:
-            self.send_error_message("job_does_not_exist", req)
+            retry_results = self._jm.retry_jobs(
+                [req.job_id] if req.job_id else req.job_id_list
+            )
+            for retry_result in retry_results:
+                self.send_comm_message("job_retried", retry_result)
+                self.send_comm_message("new_job", {"job_id": retry_result["retry_id"]})
```

Comment:
I suggest changing this to:
```
self.send_comm_message("job_retry_response", retry_results)
self.send_comm_message("new_job", { "job_id_list": job.retry_id for job in retry_results if job.retry_id })
```

i.e. the job id list for the `new_job` message contains an array of all the retries that were successful. The FE doesn't do anything useful with the `new_job` message, it just prompts a narrative save.